shared_libraries/orca_shared/reconciliation/shared_reconciliation.py

```diff
@@ -3,20 +3,22 @@
 Description: Shared library that combines common functions and classes needed for
              reconciliation operations.
 """
-import logging
 from datetime import datetime, timezone
 
 # Standard libraries
 from enum import Enum
 from typing import Optional
 
+from aws_lambda_powertools import Logger
+
 # Third party libraries
 from sqlalchemy import text
 from sqlalchemy.future import Engine
 
 from orca_shared.database import shared_db
 
-LOGGER = logging.Logger("ORCA")
```

Comment:
This is a weird edge-case. Recommend reverting for now, and adding to the card for standardizing logger propogation.